### PR TITLE
Add note to Kubernetes RBAC docs about RoleBindings and namespaces

### PR DESCRIPTION
**Pull request by jmara**
### What does this PR do?

I've updated the documentation to give the user a hint that Rolebinding per namespace in only available in version 1.5+

### Motivation

Spend an hour digging through Github Issues/PRs to find that the changes how Traefik uses the kubernetes api (namespaves) was not merged into the 1.4 branch. The current docs lead to the impression that this (RoleBinding instead of ClusterRoleBinding) should just work out of the box.

Please see #1626 and #1895

### CODE REVIEW

**Comment by timoreimann** [comment]
Thanks a lot! Could you rebase your PR against the `v1.5` branch? That way, we can ensure the upcoming release will have the note as well.

**Comment by jmara** [comment]
Hmm that doesn't look right, do you have a hint for me how to rebase correctly?

**Comment by timoreimann** [comment]
I need to rebase your commit _onto_ the `v1.5` branch.

I can also take care of it for you if you don't mind. Let me know.

**Comment by jmara** [comment]
Sure, please go ahead. I'm glad I could help.

**Comment by timoreimann** [comment]
@jmara alright, rebase complete. (By the way, I did it my doing a git rebase onto like this: `git rebase --onto upstream/v1.5 HEAD^`.)

File: docs/user-guide/kubernetes.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ If your cluster is configured with RBAC, you will need to authorize Træfik to u
 
 RoleBindings per namespace enable to restrict granted permissions to the very namespaces only that Træfik is watching over, thereby following the least-privileges principle. This is the preferred approach if Træfik is not supposed to watch all namespaces, and the set of namespaces does not change dynamically. Otherwise, a single ClusterRoleBinding must be employed.
 
+!!! note
+    RoleBindings per namespace will be available in Træfik 1.5+ please use ClusterRoleBinding for any older version.
```
Comment (timoreimann) [inline docs/user-guide/kubernetes.md]: Can we rewrite this as following:

_RoleBindings per namespace are available in Træfik 1.5 and later. Please use ClusterRoleBindings for older versions._

(Present tense; replace plus by verbatim text; second sentence; plural for ClusterRoleBinding too; older versions.)

Comment (jmara) [reply docs/user-guide/kubernetes.md]: Sure sounds good. From a documentation perspective in the 1.5 branch present tens makes totally sense. The rest is fine with me. I'll try to make the changes :)

**Pull request merged**